github_connector_odoo/models/odoo_module_version.py

```diff
@@ -391,3 +391,37 @@ def create_or_update_from_manifest(
             except Exception as e:
                 _logger.error(
                     'Unable to read the OCA icon image, error is %s' % e)
+
+    @api.model
+    def cron_clean_odoo_module_version(self):
+        module_versions = self.search([])
+        module_versions.clean_odoo_module_version()
+
+    @api.multi
+    def clean_odoo_module_version(self):
+        for module_version in self:
+            # Compute path(s) to analyze
+            branch = module_version.repository_branch_id
+            if branch.module_paths:
```

Comment:
Hi thanks. 
Not blocking point. (can be done later).
it could be great to refactor this lines because duplicated with a new function in repository.branch model named : 

    @api.multi 
    def _get_module_paths:
        self.ensure_one()
       branch = self
        if branch.module_path  
             paths = []
             for path in branch.module_paths.split('\n'):
                if path.strip():
                    paths.append(os.path.join(branch.local_path, path))
            else:
                paths = [branch.local_path]
    return paths

Reply:
Instead, I would suggest having a new functional field.

Reply:
sorry my bad it will be not possible as we might have list in paths

Reply:
As you wish on that point.
* it can be the function, (as proposed).
* it can be a new computed field my_computed_field = fields.Char(help="coma separated path") (as in ADDONS_PATH syntax), that we have just to split after.
no clear point of view on that point.

Reply:
removed redundant code and added function for that.